index.html

```diff
@@ -211,10 +211,24 @@
           <h2>LDP Containers</h2>
           <p>
             Implementations MUST support the creation and management of [[!LDP]] Containers.
-            LDP Direct Containers MUST NOT permit <code>ldp:contains</code> as their membership-predicate
-            and requests that would do so MUST fail with 409 Conflict.
+          </p>
+          <p>
+            LDP Containers MUST distinguish containment triples from membership and minimal-container triples.
+            If an implementation cannot make this distinction solely from the predicate, it MUST NOT permit
+            <code>ldp:contains</code> as the <code>membership predicate</code> for LDP Direct and Indirect
+            Containers or as a client-managed property, and requests that would do so MUST fail with 409 Conflict
+            and accompanying constraints document. Implementations that can distinguish containment triples MAY
+            permit  <code>ldp:contains</code> as a <code>membership predicate</code> or client-managed property;
+            such implementations SHOULD support <code>Prefer:</code> header values allowing clients to distinguish this data in the representation of a LDP-R as defined in [[!LDP]]:
```

Comment:
I'm open to that, but I'd like to think about it more. Do you feel like that decision is necessary to make on this PR?